### ISSUE TITLE
Dropboxes - delet source files not checked lead to lot of copies

### ISSUE THREAD

**Issue by alexolivan** [ElvishArtisan/rivendell#686]
Hi again!

I've a curious situation here: 
I have setup a Dropbox to drop a bunch of old, funny 50's ads to fake traffic/ads... fun stuff!
This time but I didn't checked the 'Delete source files after import' .... and I screwed the thing a little....What happened next is that I discoverd later, in rdlibrary, the whole 1000 cart span I set the traffic group filled with carts that are mostly copies.

I've re-check that with my other Dropboxes where the files are deleted after being imported the import keeps working well... and it does... so, unless I'm missing something in RDAdmin that controls this behaviour at group level, or whatever, I feel as it loops the files in the folder endlessly, importing them once and again and again...

(Apart of this, I would appreciate if someone could point me what's the best way to delete the whole group content, deleting 1000 carts one by one would be painfully slow.... I guess there's probably some CLI/RML workaround but I'm not finding it right now)
 
Best regards!

**Comment by ElvishArtisan** [comment]
> (Apart of this, I would appreciate if someone could point me what's the best way to delete the whole group content, deleting 1000 carts one by one would be painfully slow.... I guess there's probably some CLI/RML workaround but I'm not finding it right now)

That one's easy! First, start ```rdlibrary(1)``` and make sure that the **Allow Cart Dragging** checkbox is un-ticked. Then, select all of the carts that you wish to delete and touch the **Delete** button. Touch the **Yes** button in the confirmation dialog and you're done.

**Comment by ElvishArtisan** [comment]
Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed in 2eebe728. Please test!

**Comment by alexolivan** [comment]
Great! will report as soon as I can!
Now I've no qt5 build in hand (Soon I'll have a more powerful machine to have a parallel qt5 install to join the beta test!)

Thank you very much!

**Comment by ElvishArtisan** [comment]
Bump! Did this get fixed?

**Comment by alexolivan** [comment]
Hi! Until these weekend I didn't had a qt5-branch testbed to test all this (I was on master 3.6)... Today I have setup Dropboxes and I have successfully ingested one single tune, that was deleted after import normally... I didn't recall on this bug as opened (though other's experiences could confirm the fix works... I'm sure it is indeed fixed) so I wasn't even thinking on this as I did it.

Anyhow, tomorrow ( in the next days ) I'm going to make extensive use of the Dropboxes I've setup today... Since I'm re-doing from scratch the former 3.6 setup, this means hundreds of tunes to import, so I thing it will be a comprehensive test.
